docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -229,64 +229,49 @@ IMPORTANT: {deprecation-notice-installation}
 [float]
 ==== Elastic Cloud
 
-If you're running APM Server in Elastic cloud, you can configure your own user settings right in the Elasticsearch Service Console.
-Any changes are automatically appended to the `apm-server.yml` configuration file for your instance.
-
-Full details are available in the {cloud}/ec-manage-apm-settings.html[APM user settings] documentation.
+If you're running APM Server in Elastic cloud, configure your user settings in the {ess} Console.
+See {cloud}/ec-manage-apm-settings.html[APM user settings] for more information.
 
 [float]
 ==== Self installation
 
-You can edit the `apm-server.yml` configuration file to customize it to your needs.
-The location of this file varies by platform, but the <<directory-layout>> will help you locate it.
-All available configuration options are outlined in
-{apm-server-ref-v}/configuring-howto-apm-server.html[configuring APM Server].
+Starting in version 8.0.0, {fleet} uses to the APM integration to set up and manage APM index templates,
+ILM policies, and ingest pipelines. APM Server will only connect to {es} _after_ the APM integration has been installed.
```

Comment:
This needs to be mentioned somewhere prominent. If users do not follow this, the APM Server generally starts up but eventually fills up the internal queue. Until then the issue is only observable through logs and missing data in the APM UI.

Reply:
👍 Made more prominent and will add (more prominently) to release notes as well.